dist/css/jsonforms.css

```diff
@@ -1,10 +1,5 @@
 .jsf {
-  /*!
- * Bootstrap v3.3.5 (http://getbootstrap.com)
- * Copyright 2011-2015 Twitter, Inc.
- * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
- */
-  /*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
```

Comment:
You somehow generated differently from the last changes. Can you check if you or the previous committer made a mistake?